### PR TITLE
Fix didTimeout definition

### PR DESCRIPTION
**Pull request by rmcilroy**
Addresses #61

### CODE REVIEW

**Comment by rmcilroy** [comment]
@annevk, @igrigorik please take a look, thanks.

File: index.html [hunk 1]
```diff
@@ -442,8 +442,13 @@
...
       </ol>
-      <p>The <code><dfn>didTimeout</dfn></code> attribute on an `IdleDeadline`
-      object MUST return `true` if the callback was invoked by the <a>invoke idle callback timeout algorithm</a>, and `false` otherwise.</p>
+      <p>Each <a>IdleDeadline</a> has an associated <code><dfn>timeout</dfn>
+      </code>, which is initially false. The <code><dfn>
+      didTimeout</dfn></code> getter MUST return <a>timeout</a>. The <a>invoke
+      idle callback timeout algorithm</a> sets <a>timeout</a> to true to specify
+      that the callback is being executed outside an idle period, due to it
+      having exceeded the timeout property passed as an <a>IdleRequestOptions</a>
+      parameter when the callback was registered.</p>
```
Comment (annevk) [inline index.html]: This last sentence would be better as a note, to clearly indicate it's informative.

File: index.html [hunk 2]
```diff
@@ -442,8 +442,13 @@
         <li>If <var>timeRemaining</var> is negative, set it to 0.</li>
         <li>Return <var>timeRemaining</var>.</li>
       </ol>
-      <p>The <code><dfn>didTimeout</dfn></code> attribute on an `IdleDeadline`
-      object MUST return `true` if the callback was invoked by the <a>invoke idle callback timeout algorithm</a>, and `false` otherwise.</p>
+      <p>Each <a>IdleDeadline</a> has an associated <code><dfn>timeout</dfn>
```
Comment (annevk) [inline index.html]: I don't think you want `<code>` here. It's just a concept after all.

**Review by annevk — APPROVED** [review]
LGTM with nits.

File: index.html (hunk 2 above)
Comment (rmcilroy) [reply index.html]: Done

File: index.html (hunk 1 above)
Comment (rmcilroy) [reply index.html]: Done

**Comment by rmcilroy** [comment]
Thanks, comments addressed.

File: index.html [hunk 3]
```diff
@@ -442,8 +442,14 @@
...
         <li>Return <var>timeRemaining</var>.</li>
       </ol>
-      <p>The <code><dfn>didTimeout</dfn></code> attribute on an `IdleDeadline`
-      object MUST return `true` if the callback was invoked by the <a>invoke idle callback timeout algorithm</a>, and `false` otherwise.</p>
+      <p>Each <a>IdleDeadline</a> has an associated <dfn>timeout</dfn>, which is
+      initially false. The <code><dfn>didTimeout</dfn></code> getter MUST
+      return <a>timeout</a>.</p>
+      <p class='note'>The <a>invoke idle callback timeout algorithm</a> sets
+      <a>timeout</a> to true to specify that the callback is being executed
+      outside an idle period, due to it having exceeded the timeout property
```
Comment (igrigorik) [inline index.html]: With this update we now have two "timeout" concepts being using interchangeably. Let's make them more explicit in text to minimize confusion?

..."due to it having exceeded the value of the `<a>IdleDeadline</a>'s <a>timeout</a>` attribute"?

Comment (rmcilroy) [reply index.html]: Done (it was <a>IdleRequestOptions</a>'s timeout property though).

**Review by igrigorik — APPROVED** [review]
👍

**Pull request merged**